docs/manuals/gui/styling/index.md

```diff
@@ -43,29 +48,59 @@ font size, you could write:
 And all pages will appear twice as big.
 
 
-### Markdown styles
+### Styling Markdown
 
-Thanks to the [_Attribute Lists_](https://python-markdown.github.io/extensions/attr_list/)
+Thanks to the [*Attribute Lists*](https://python-markdown.github.io/extensions/attr_list/)
 extension, the Markdown text can hold attributes used for styling.
 
 If, for example, your Markdown content is the following:
 ```
-...
-This line should be displayed in blue.
-{ .blue-line }
-...
+Un-styled text line.
+
+Styled text line.
+{: .blue }
 ```
 
-and a style sheet used by the application indicates:
+and a stylesheet used by the application defined the *blue* CSS class:
 ```css
-.blue-line {
+.blue {
   color: blue;
 }
 ```
-then the text line is displayed in blue.
+then the text line is displayed in blue:
+
+<figure>
+  <img src="attribute-para-d.png" class="visible-dark" />
+  <img src="attribute-para-l.png" class="visible-light"/>
+  <figcaption>Styling paragraphs</figcaption>
+</figure>
+
+That would apply to the entire preceding paragraph.
+
+If you want to set the style for an inline element of the text (such as a
+bold section identified by the "\*\*" markers), you can use the same
+attribute syntax, right after the text fragment, without any space
+character.<br/>
+The following Markdown content:
+```
+A text that has a **specific**{: .blue } style.
+```
+will apply the *blue* class to the bold text fragment:
+<figure>
+  <img src="attribute-fragment-d.png" class="visible-dark" />
+  <img src="attribute-fragment-l.png" class="visible-light"/>
+  <figcaption>Styling fragments</figcaption>
+</figure>
+
+Note to style a heading, the syntax is slightly different: right after the text of the heading,
+you must insert a white space, as many sharp (#) characters as the heading uses, another white
+space, then the style construct.<br/>
```

Comment:
"then the _Markdown Attribute Lists_ construct" would be more exact, but if that’s fine by you, it is by me as well.